include/slang-rhi.h

```diff
@@ -2642,6 +2642,12 @@ class IHeap : public ISlangUnknown
     virtual SLANG_NO_THROW Result SLANG_MCALL removeEmptyPages() = 0;
 };
 
+struct HeapReport
```

Comment:
Ah, just realized we now have `HeapReport` and `IHeap::Report` .. this is a bit confusing IMO